piccolo/apps/asgi/commands/new.py

```diff
@@ -10,10 +10,18 @@
 
 TEMPLATE_DIR = os.path.join(os.path.dirname(__file__), "templates/app/")
 SERVERS = ["uvicorn", "Hypercorn"]
-ROUTERS = ["starlette", "fastapi", "blacksheep", "litestar", "esmerald"]
+ADMIN_DEPENDENCIES = ["piccolo_admin>=1.0.0"]
 ROUTER_DEPENDENCIES = {
-    "fastapi": ["fastapi>=0.100.0"],
+    "starlette": ["starlette", *ADMIN_DEPENDENCIES],
+    "fastapi": ["fastapi>=0.100.0", *ADMIN_DEPENDENCIES],
+    "blacksheep": ["blacksheep", *ADMIN_DEPENDENCIES],
+    "litestar": ["litestar", *ADMIN_DEPENDENCIES],
+    # We don't currently use Piccolo Admin with esmerald, due to dependency
+    # conflicts with FastAPI. We'll revisit this at some point in the future
+    # when newer versions of FastAPI are released (DT - 12th December 2023).
+    "esmerald": ["esmerald"],
 }
```

Comment:
I decided to omit `piccolo_admin` from `esmerald` this way instead of having this in `requirements.txt`:

```jinja
{% if "esmerald" not in router_dependencies %}
piccolo_admin
{% endif %}
```

Because at some point I'd like to version pin the routers a bit more. So it might be that we have "esmerald>X.X.X" in which case the `if` statement would fail.